nimx/cursor.nim

```diff
@@ -7,6 +7,7 @@ when defined(js) or defined(emscripten):
 elif defined(appKit):
     import private.objc_appkit
     enableObjC()
+    import sdl2
```

Comment:
This fixes the following error:
`/Users/matthew/.nimble/pkgs/nimx-0.1/nimx/cursor.nim(70, 42) Error: undeclared identifier: 'SystemCursor'`